src/components/List.js

```diff
@@ -79,6 +80,21 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  // Confirm that the user would like to delete this item
+  const confirmDelete = (item) => {
+    const confirmed = window.confirm(
+      'Are you sure you want to delete this item?',
+    );
+    if (confirmed) {
+      try {
+        db.collection(token).doc(item.id).delete();
+      } catch (error) {
+        setDeleteError(`Problem deleting ${item.itemName}`);
```

Comment:
Might be a good idea to clear this error somewhere, if, for example, there ends up being an error and you make it to the catch block.

Reply:
Maybe to keep the same strategy for now you could just alert the error, then you wouldn't need to clear it from the page, really.

Reply:
The second options sounds like a good short term fix and then if we're able we could tackle doing it the way we have it now, but then clearing it or resetting it to an empty string... @DeeDowns , what do you think?